web/src/Routes.tsx

```diff
@@ -2,63 +2,76 @@ import * as React from 'react'
 
 import SwaggerUI from 'swagger-ui-react'
 import { Routes as Switch, Route } from 'react-router-dom'
+import {
+    BillingHome,
+    BillingSeqrProp,
+    BillingCostByTime,
+    BillingInvoiceMonthCost,
+} from './pages/billing'
 import DocumentationArticle from './pages/docs/Documentation'
 import SampleView from './pages/sample/SampleView'
 import FamilyView from './pages/family/FamilyView'
 import ProjectSummaryView from './pages/project/ProjectSummary'
 import ProjectsAdmin from './pages/admin/ProjectsAdmin'
 import ErrorBoundary from './shared/utilities/errorBoundary'
 import AnalysisRunnerSummary from './pages/project/AnalysisRunnerView/AnalysisRunnerSummary'
-import BillingDashboard from './pages/billing/BillingDashboard'
 
 const Routes: React.FunctionComponent = () => (
     <Switch>
-        <Route path="/documentation/:id?" element={<DocumentationArticle />} />
-
-        <Route path="/swagger" element={<SwaggerUI url="/openapi.json" tryItOutEnabled={true} />} />
+        <Route path="/" element={<DocumentationArticle articleid="index" />} />
 
+        <Route path="admin" element={<ProjectsAdmin />} />
         <Route
-            path="/analysis-runner/:projectName?"
+            path="/project/:projectName?/:page?"
             element={
                 <ErrorBoundary>
-                    <AnalysisRunnerSummary />
+                    <ProjectSummaryView />
                 </ErrorBoundary>
             }
         />
-
         <Route
-            path="/project/:projectName?/:page?"
+            path="project/:projectName/participant/:participantName"
             element={
                 <ErrorBoundary>
-                    <ProjectSummaryView />
+                    <SampleView />
                 </ErrorBoundary>
             }
         />
 
-        <Route path="admin" element={<ProjectsAdmin />} />
-
-        <Route path="/" element={<DocumentationArticle articleid="index" />} />
-
         <Route
-            path="sample/:sampleName/:sequencingGroupName?"
+            path="/analysis-runner/:projectName?"
             element={
                 <ErrorBoundary>
-                    <SampleView />
+                    <AnalysisRunnerSummary />
                 </ErrorBoundary>
             }
         />
 
+        <Route path="/billing/" element={<BillingHome />} />
```

Comment:
We should (later) setup the path routes for billing, to avoid this function getting too cluttered

Reply:
Sounds good